cmd/soroban-rpc/internal/db/db.go

```diff
@@ -293,6 +305,10 @@ func (w writeTx) Commit(ledgerSeq uint32) error {
 		return err
 	}
 
+	if err := w.eventWriter.trimEvents(ledgerSeq, w.ledgerRetentionWindow); err != nil {
```

Comment:
I just wanna make sure that you've at least tested locally that trimming works (e.g. 15min retention) :pray:

Reply:
Okay I tested it locally, and it is working fine